integration-tests/process.go

```diff
@@ -0,0 +1,75 @@
+package integrationtests
+
+import (
+	"strings"
+	"strconv"
+	"fmt"
+)
+
+type ProcessInfo struct {
+	Name string
+	ExePath string
+	Uid int
+	Gid int
+	Pid int
+	Args string
+}
+
+func NewProcessInfo(line string) (*ProcessInfo, error) {
+	parts := strings.SplitN(line, ":", 6)
+
+	if len(parts) != 6 {
+		return nil, fmt.Errorf("not enough parts to gRPC process info string: %s", line)
```

Comment:
I think I like the error message on `NewProcessLineage` a little more, could we rewrite this one?
```suggestion
		return nil, fmt.Errorf("invalid gRPC string for process info: %s", line)
```